lambda/service/handler/get_account_handler.go

```diff
@@ -0,0 +1,64 @@
+package handler
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/pennsieve/account-service/service/models"
+	"github.com/pennsieve/account-service/service/store_dynamodb"
+)
+
+func GetAccountHandler(request events.APIGatewayV2HTTPRequest) (events.APIGatewayV2HTTPResponse, error) {
+	handlerName := "GetAccountHandler"
+	uuid := request.PathParameters["id"]
+	ctx := context.Background()
```

Comment:
I don't think this is a big deal, since we aren't really doing anything with the Context now, but I think ideally the signature of `AccountServiceHandler` would have a Context as it's first arg. Then the Lambda runtime would pass it in for us, that Context could be passed through `router.Start()` to the sub-handlers instead of creating a new one in each handler.

I recently updated the template for serverless services to reflect this, but it wasn't there when this service was started.

Reply:
@jfromme oh yes, thats exactly what I do in the integration service, however I used the template and it didn't have it, so I wasnt sure if you were using an updated signature. I meant to follow up on it,  but didnt get the chance.

https://github.com/Pennsieve/integration-service/blob/main/lambda/service/handler/handler.go#L23

Reply:
Updated